demo/app.py

```diff
@@ -106,15 +156,37 @@
 # Collect parameters from the contract editor and store them in a dict.
 @callback(
     Output("ctr-params", "data"),
+    Output("option-type-container", "style"),
```

Comment:
Consider using `set_props`, which is a new feature,
so that you don't have to add output for each property.

See https://community.plotly.com/t/dash-2-17-0-released-callback-updates-with-set-props-no-output-callbacks-layout-as-list-dcc-loading-trace-zorder/84343/4